app/main/views/suppliers.py

```diff
@@ -362,6 +362,37 @@ def do_search(search_query, offset, result_count, new_domains, framework_slug):
     except (KeyError, IndexError):
         terms = {}
 
+    approved_categories = (
+        db
+        .session
+        .query(
+            SupplierDomain.supplier_id,
+            func.count(SupplierDomain.id).label('total')
+        )
+        .filter(
+            SupplierDomain.status == 'assessed',
+            SupplierDomain.price_status == 'approved'
+        )
+        .group_by(SupplierDomain.supplier_id)
+        .subquery()
+    )
+
+    registered_sellers = (
+        db
+        .session
+        .query(Supplier.id)
+        .join(
+            approved_categories, approved_categories.columns.supplier_id == Supplier.id, isouter=True
+        )
+        .filter(
+            or_(
```

Comment:
Would this additional filter show hybrids and recruiters?
```suggestion
            Supplier.data['recruiter'].astext == 'no',
            or_(
```

Reply:
Yes this seems to be correctly exclude hybrids/recruiters from sellers to filter out.